server/src/main/java/io/unitycatalog/server/persist/TableRepository.java

```diff
@@ -284,10 +268,10 @@ public ListTablesResponse listTables(
     Integer pageSize = getPageSize(maxResults);
     String hql =
         "FROM TableInfoDAO t WHERE t.schemaId = :schemaId and "
-            + "(t.updatedAt < :pageToken OR :pageToken is null) order by t.updatedAt desc";
+            + "(t.name > :pageToken OR :pageToken is null) order by t.name asc";  
```

Comment:
@haogang is this how we wish implement list table pagination for all entities (name in ascending order) ?